Applications/TextEditor/TextEditorWidget.cpp

```diff
@@ -651,6 +668,13 @@ void TextEditorWidget::update_html_preview()
     m_page_view->scroll_into_view(current_scroll_pos, true, true);
 }
 
+void TextEditorWidget::update_GML_preview()
+{
+    auto current_scroll_pos = m_page_view->visible_content_rect();
+    m_page_view->load_from_gml(m_editor->text());
```

Comment:
`m_page_view` is a web view, we should not be inserting random children into it.
This should be done with an empty `GUI::Widget` at the same level as `m_page_view`, possibly with both in a `GUI::StackWidget`.